ui/src/app/shared/components/object-editor/object-editor.tsx

```diff
@@ -91,10 +78,24 @@ export const ObjectEditor = <T extends any>({type, value, buttons, onChange}: Pr
                         renderIndentGuides: false,
                         scrollBeyondLastLine: true
                     }}
-                    onChange={() => setIsModified(true)}
+                    onChange={v => {
+                        try {
+                            onChange(parse(v));
+                            setError(null);
+                        } catch (e) {
+                            setError(e);
+                        }
+                    }}
                 />
             </div>
             <div style={{paddingTop: '1em'}}>
+                {error && (
```

Comment:
move error to below editor so when they pop-up, editor does not jump down

Reply:
It's frustrating when sites/apps don't consider these pop-ups and you end up clicking the wrong thing because the whole page shifts